returnn/datasets/generating.py

```diff
@@ -938,6 +938,14 @@ def get_data_dtype(self, key):
     """
     return str(self.data[0][key].dtype)
 
+  def is_data_sparse(self, key):
+    """
+    :param str key:
+    :rtype: bool
+    """
+    data_type = self.get_data_dtype(key)
+    return data_type.startswith("int")
```

Comment:
I guess no need to check for "uint" etc., right?

Reply:
Why not?